psalm.xml.dist

```diff
@@ -12,6 +12,7 @@
         <ignoreFiles>
             <directory name="vendor"/>
             <file name="src/NewDOMSerializer.php" />
+            <file name="src/DOMSerializerPointer.php" />
```

Comment:
@hanspagel I think the two files can be completely removed? Do they serve some future ideas?